src/secp256k1/src/modules/recovery/main_impl.h

```diff
@@ -126,6 +126,7 @@ int secp256k1_ecdsa_sign_recoverable(const secp256k1_context* ctx, secp256k1_ecd
     int recid;
     int ret = 0;
     int overflow = 0;
+    const unsigned char secp256k1_ecdsa_recoverable_algo16[17] = "ECDSA+Recovery  ";
```

Comment:
why size 17 and not 16? (the given string is 16 chars)
```
>>> len("ECDSA+Recovery  ")
16
```

Reply:
null terminator